src/hotspot/share/compiler/compileBroker.cpp

```diff
@@ -87,7 +87,10 @@
 #include "opto/c2compiler.hpp"
 #endif
 #ifdef JEANDLE
+#pragma push_macro("AARCH64")
```

Comment:
Just like we discussed before, perhaps we can solve this issue together with the assert macro problem through a separate header file.

Reply:
I have created #52 to track the issue. Let's do the refactoring in a separate PR.